app/models/manageiq/providers/hawkular/middleware_manager/middleware_diagnostic_report.rb

```diff
@@ -0,0 +1,139 @@
+require 'timeout'
+
+module ManageIQ::Providers
+  class Hawkular::MiddlewareManager::MiddlewareDiagnosticReport < ApplicationRecord
+    self.table_name = 'middleware_diagnostic_reports'
+
+    STATUS_QUEUED = 'Queued'.freeze
+    STATUS_RUNNING = 'Running'.freeze
+    STATUS_ERROR = 'Error'.freeze
+    STATUS_READY = 'Ready'.freeze
```

Comment:
This also would lead some changes to UI PR, because is't displaying the status as is: https://github.com/ManageIQ/manageiq-ui-classic/pull/1764/files#diff-002598c1d95530eae2e85c719166439fR63